### DISCUSSION TITLE
内存规则组提示已损坏应该如何解决

### DISCUSSION THREAD

**Discussion by Aliyuo** [gkd-kit/gkd#406] [General]
![6aa4e31b1f8553d6b90dd9e99f22dee](https://github.com/gkd-kit/gkd/assets/147463585/7498f3ab-4270-4c3d-a343-2f1d11646d21)

**Comment by Aliyuo** [comment]
使用选择执行器可以消除广告

**Reply by lisonge** [reply]
> 已损坏应该如何解决

你的规则语法写错了，所以会显示损坏
